src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
+        <nav class="header__nav nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
```

Comment:
add an empty line between multiline siblings for better readability
```suggestion
            </li>
            
            <li class="nav__item">
```